contrib/avro-cmake/CMakeLists.txt

```diff
@@ -62,9 +62,12 @@ target_link_libraries (_avrocpp PRIVATE ch_contrib::snappy)
 
 # create a symlink to include headers with <avro/...>
 set(AVRO_INCLUDE_DIR "${CMAKE_CURRENT_BINARY_DIR}/include")
-ADD_CUSTOM_TARGET(avro_symlink_headers ALL
-    COMMAND ${CMAKE_COMMAND} -E make_directory "${AVRO_INCLUDE_DIR}"
-    COMMAND ${CMAKE_COMMAND} -E create_symlink "${AVROCPP_ROOT_DIR}/api" "${AVRO_INCLUDE_DIR}/avro"
+ADD_CUSTOM_COMMAND(OUTPUT "${AVRO_INCLUDE_DIR}"
+        COMMAND ${CMAKE_COMMAND} -E make_directory "${AVRO_INCLUDE_DIR}"
```

Comment:
But we don't need it (symlinks) at all...

Reply:
You are absolutely right, and there is no better way to remove a bug than to remove the code.